### PR TITLE
Change CopyToPublishDirectory on Content items instead of explicitly …

### LINKED ISSUE #2295: Publishing a project with UI class library publishes cshtml files from class library
The issue crops up regardless of `CopyRazorGenerateFilesToPublishDirectory`  setting. When publishing, Razor SDK removes cshtml files from the current project from being published, but it does not prevent views from referenced UI class libraries from being published. 

Relevant code: https://github.com/aspnet/Razor/blob/dev/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets#L504-L510

### PR DESCRIPTION
**Pull request by pranavkm**
…removing this from publish items

Fixes #2295

### CODE REVIEW

**Comment by pranavkm** [comment]
@ajaybhargavb any idea why the test's failing on Win x64?

**Comment by ajaybhargavb** [comment]
@pranavkm I replied here https://github.com/aspnet/Razor/issues/2303#issuecomment-384090144.

**Comment by pranavkm** [comment]
@dotnet-bot test Windows Release x64 Build please

**Comment by pranavkm** [comment]
@dotnet-bot test Windows Release x64 Build please

**Comment by pranavkm** [comment]
Verifying this change, it doesn't look like this works correctly. The `<Content Update="<anything here>" />` ends up updating all content items. Working on figuring out a solution. If I don't hear anything concrete, we should punt this to 2.2.0.

File: src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets [hunk 1]
```diff
@@ -283,6 +290,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ItemGroup Condition="'$(EnableDefaultRazorGenerateItems)'=='true'">
       <RazorGenerate Include="@(Content)" Condition="'%(Content.Extension)'=='.cshtml'" />
     </ItemGroup>
+
+    <!-- Disable publishing RazorGenerate items if CopyRazorGenerateFilesToPublishDirectory=false -->
+    <ItemGroup Condition="'$(CopyRazorGenerateFilesToPublishDirectory)'=='false' and '$(ResolvedRazorCompileToolset)'=='RazorSdk' and '$(RazorCompileOnPublish)'=='true'">
+      <Content Update="@(RazorGenerate)" CopyToPublishDirectory="Never" />
```
Comment (rynowak) [inline src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets]: Does this need a condition?

**Comment by pranavkm** [comment]
@rynowak updated this based on our email conversation. We can chat about this when you're in.

File: src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets [hunk 2]
```diff
@@ -305,6 +312,20 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ItemGroup Condition="'$(EnableDefaultRazorGenerateItems)'=='true'">
       <RazorGenerate Include="@(Content)" Condition="'%(Content.Extension)'=='.cshtml'" />
     </ItemGroup>
+
+    <!--
+      Ideally we want to able to update all Content items that also appear in RazorGenerate to have
```
Comment (pranavkm) [inline src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets]: Updated comment

**Pull request closed without merging**